tests/unit/backend/layers/api/test_curation_api.py

```diff
@@ -934,19 +975,25 @@ def test_get_colletion_with_dataset_no_metadata(self):
     def _test_response(self, collection_version: CollectionVersion, status_code=200, auth=True) -> dict:
         version_id = collection_version.version_id
         collection_id = collection_version.collection_id
+        is_open_revision = (
+            collection_version.published_at is None
+            and collection_version.canonical_collection.originally_published_at is not None
```

Comment:
why would you reuire the canonical entity to have a `originally_published_at` value of `None` in order to say it is an open revision? This seems incorrect. Am I missing a hack?

Reply:
it checks for `is not None`